solutions/solution-basic/src/main/resources/microservice.yaml

```diff
@@ -41,8 +41,15 @@ servicecomb:
   accesslog:
     enabled: true
   metrics:
+    window_time: 60000
```

Comment:
why not use **windowTime**  instead.  maybe there needs a unified style here

Reply:
not sure which style is better, maybe unify them in the furture

Reply:
ok.